javaslang/src/main/java/javaslang/collection/Stream.java

```diff
@@ -6,7 +6,8 @@
 package javaslang.collection;
 
 import javaslang.*;
-import javaslang.collection.Stream.*;
+import javaslang.collection.Stream.Cons;
+import javaslang.collection.Stream.Empty;
```

Comment:
(Mhh, we seem to use different formatters - we should unify them. I like the wildcards)